.htaccess

```diff
@@ -28,6 +28,14 @@ DirectoryIndex index.php index.html index.htm
   php_value max_input_vars                  10007
   php_flag mbstring.encoding_translation    off
 </IfModule>
+# Support PHP7 as well as PHP5.
+<IfModule mod_php7.c>
+  php_flag session.auto_start               off
+  php_value mbstring.http_input             pass
+  php_value mbstring.http_output            pass
+  php_value max_input_vars                  10007
+  php_flag mbstring.encoding_translation    off
+</IfModule>
```

Comment:
This section should be moved before the php5 block, as per @quicksketch's feedback on the issue.